test/unit/id_resolver_test.rb

```diff
@@ -190,7 +190,7 @@
             { 'id' => 85, 'name' => 'git::params', 'created_at' => '2015-01-27T07:24:57.306Z', 'updated_at' => '2015-01-27T07:24:57.306Z' }
           ] })
 
-        resolver_run.call.must_equal [70, 27, 85]
+        resolver_run.call.map { |o| o['id'] }.flatten.must_equal [70, 27, 85]
```

Comment:
This looks like inconsistent change of the interface. Why did you change it to return array of hashes?